api/jobs_test.go

```diff
@@ -754,7 +803,7 @@ func TestPatchJobStatusHandler(t *testing.T) {
 
 		Convey("When a patch request is made with valid job ID and valid patch operations", func() {
 			req := httptest.NewRequest("PATCH", fmt.Sprintf("http://localhost:25700/jobs/%s", validJobID1), bytes.NewBufferString(validPatchBody))
-			headers.SetIfMatch(req, eTagValidJobID1)
```

Comment:
You need to remove the const `eTagValidJobID1 ` at the top of this file. Just search for `eTagValidJobID1 `